app/forms/users/name_form.rb

```diff
@@ -5,7 +5,7 @@ class NameForm < BaseForm
     validates :first_name, :last_name, presence: true
 
     def save
-      user.update(first_name: first_name, last_name: last_name) if valid?
+      user.update!(first_name: first_name, last_name: last_name) if valid?
```

Comment:
I actually really like this one! It is too easy to miss using the bang version. This is the sort of thing that makes using rubocop worthwhile IMO